realsense_ros_camera/launch/includes/nodelet.launch.xml

```diff
@@ -42,6 +42,8 @@
   <arg name="enable_pointcloud"   default="false"/>
   <arg name="enable_sync"         default="false"/>
 
+  <arg name="base_frame_id"       default="realsense"/>
```

Comment:
I think this arg should be renamed to something like `camera`, `name`, `prefix` or a combination of them.